data/helpers.d/network

```diff
@@ -0,0 +1,9 @@
+# Check availability of a web path
+#
+# example: ynh_path_validity $domain$path
+#
+# usage: ynh_path_validity $domain_and_path
+# | arg: domain_and_path - complete path to check
+ynh_path_validity () {
```

Comment:
Imho this name is confusing : 
- path alone is ambiguous and might be interpreted as a filesystem path (like `/etc/toto`)
- validity might be interpreted as "syntaxically valid"

I would suggest something like `ynh_url_path_available`